source/.res/replace.txt

```diff
@@ -1,47 +1,44 @@
-.. -*- mode: rst -*-
 .. |product| replace:: PMM
```

Comment:
This file was cleaned up before being used to automate the replacement of replacements.